coalib/bearlib/aspects/__init__.py

```diff
@@ -21,7 +22,7 @@
 __all__ = ['Root', 'Taste', 'TasteError',
            'aspectclass', 'aspectbase', 'AspectTypeError',
            'AspectNotFoundError', 'MultipleAspectFoundError',
-           'AspectList']
+           'AspectList', 'map_setting_to_aspect']
```

Comment:
add trailing comma's and put the final `]` on a new line so it doesnt need updating all the time.